model/task/task_test.go

```diff
@@ -3773,6 +3773,72 @@ func TestByExecutionTasksAndMaxExecution(t *testing.T) {
 	})
 }
 
+func TestFindTaskOnPreviousCommit(t *testing.T) {
+	require.NoError(t, db.ClearCollections(Collection, OldCollection))
```

Comment:
I'm pretty sure it's only necessary to clear the `tasks` collection and not the `old_tasks` collection because this test doesn't involve archiving tasks.